sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -390,6 +402,26 @@ void ExtrinsicReflectorBasedCalibrator::trackingRequestCallback(
   RCLCPP_INFO(this->get_logger(), "New converged detections: %d", current_new_tracks_);
 }
 
+void ExtrinsicReflectorBasedCalibrator::deleteTrackRequestCallback(
+  __attribute__((unused)) const std::shared_ptr<std_srvs::srv::Empty::Request> request,
+  __attribute__((unused)) const std::shared_ptr<std_srvs::srv::Empty::Response> response)
+{
+  using std::chrono_literals::operator""s;
```

Comment:
Is this used here?

Reply:
After moving the sleep in this function. It is used now.